src/plugin/regions.js

```diff
@@ -98,15 +98,16 @@ class Region {
     }
 
     /* Play the audio region. */
-    play() {
-        this.wavesurfer.play(this.start, this.end);
+    play(start) {
```

Comment:
can you properly document this method (with @param etc)?

Reply:
I documented these, but I couldn't figure out how to make `esdoc` actually generate anything about the methods of the`Region` class -- it seems it only bothers documenting the exported classes in a module?  Or at least that's my best guess...